contracts/StakeLockContract.sol

```diff
@@ -19,7 +19,8 @@ contract StakeLockContract {
         uint256 _amount;
         uint256 _blockNum;
         uint256 _epochNum;
-        uint256 _continueCount;
+        uint256 _lockPeriodCount;  // The number of locking period the token will be locked.
+        uint256 _index;  // The index in the addressList
```

Comment:
nit: alignment.
btw: better to add some comments to all the fields in this struct.

Reply:
Done.